src/ship.h

```diff
@@ -58,6 +58,7 @@ struct Ship FINAL : public SpecializedVehicle<Ship, VEH_SHIP> {
 };
 
 static const uint SHIP_MAX_ORDER_DISTANCE = 130;
+TrackBits GetTileShipTrackStatus(TileIndex tile);
```

Comment:
I do not think this is the right place for this entry. If you look at other classes, you see they are in different files. Did you think about the place of this line, or just used the first that worked?

Reply:
I still believe ship.h is the right place for it. I looked at water.h, track_func.h, landscape.h and ship.h. Makes better sense in ship.h.